### PR TITLE
feature/add-metadata-doc-to-db

### LINKED ISSUE #117: Add basic instance metadata collection to infrastructure stack
### Feature Description

_A clear and concise description of the feature you're requesting._

Partially discussed in https://github.com/CouncilDataProject/cdp-roadmap/issues/3, it would be nice to have a collection with a single document to pull metadata about the instance.

### Use Case

_Please provide a use case to help us understand your request in context._

Most importantly would be which cdp version the instance is currently on. This would help in determine which features the instance supports.

### Solution

_Please describe your ideal solution._

Use pulumi gcp firestore document resource to create / update the same document every time the infrastructure is upgraded.
https://www.pulumi.com/docs/reference/pkg/gcp/firestore/document/

@tohuynh @isaacna can you think of other metadata that would be good to store in the database itself about the instance?
version, some subset of cookiecutter params maybe? anything else?

### PR DESCRIPTION
**Pull request by JacksonMaxfield**
### Link to Relevant Issue

This pull request resolves #117 

### Description of Changes

_Include a description of the proposed changes._

Adds basic metadata collection and a configuration doc in said collection.
image

### CODE REVIEW

File: cdp_backend/infrastructure/cdp_stack.py [hunk 1]
```diff
@@ -1,20 +1,34 @@
...
+    county_council = "county council"
+    school_board = "school board"
+    other = "other"
+
+
 class CDPStack(pulumi.ComponentResource):
     def __init__(
         self,
         gcp_project_id: str,
+        municipality_name: str = "dev",
```
Comment (isaacna) [inline cdp_backend/infrastructure/cdp_stack.py:27]: should we make the default something like `"dev_municipality"`? Very minor but just to distinguish the default values between fields

**Review by isaacna — APPROVED** [review]
Looks good to me!

**Review by tohuynh — APPROVED** [review]
Looks good to me.

File: cdp_backend/infrastructure/cdp_stack.py (hunk 1 above)
Comment (JacksonMaxfield) [reply cdp_backend/infrastructure/cdp_stack.py]: Done!

**Pull request merged**